### PR TITLE
#712 - Add npmProxyIT

### LINKED ISSUE #712: NPM proxy integration tests
Add integration tests that verify that NPM proxy can install packages

### PR DESCRIPTION
**Pull request by genryxy**
Closes #712 
Now test is disabled. See #738

### CODE REVIEW

File: src/test/java/com/artipie/npm/NpmProxyITCase.java [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+                "http://localhost:%d/%s/@hello/simple-npm-project/-/@hello/simple-npm-project-1.0.1.tgz", this.origin.port(), NpmProxyITCase.ORIGIN
+            )
+        ).openConnection();
+        con.setRequestMethod("GET");
+        con.setDoOutput(true);
+        MatcherAssert.assertThat(
+            "Verify that tgz file exists. Response status is 200",
+            con.getResponseCode(),
+            new IsEqual<>(Integer.parseInt(RsStatus.OK.code()))
+        );
```
Comment (olegmoz) [inline src/test/java/com/artipie/npm/NpmProxyITCase.java:115]: @genryxy not sure why do we need this check. The test checks that `npm install` works later, this check does not look to be required

**Review by olegmoz — CHANGES_REQUESTED** [review]
@genryxy thanks, please see one comment

File: src/test/java/com/artipie/npm/NpmProxyITCase.java (hunk 1 above)
Comment (genryxy) [reply src/test/java/com/artipie/npm/NpmProxyITCase.java:115]: @olegmoz thanks, I wanted to show by this check that the file can actually be obtained from origin repo. Probably I should remove this check or write about removing in the created issue related to enabling test.

Comment (olegmoz) [reply src/test/java/com/artipie/npm/NpmProxyITCase.java:115]: @genryxy if it helps to diagnose the problem, then maybe it deserves to be separate test, which is enabled.

Comment (genryxy) [reply src/test/java/com/artipie/npm/NpmProxyITCase.java:115]: @olegmoz it rather checks that the test is written correctly, but does not help to diagnose the problem

Comment (olegmoz) [reply src/test/java/com/artipie/npm/NpmProxyITCase.java:115]: @genryxy still looks like it should be moved to another test. So we will have test that checks Artipie server may be used as target for `npm install` command and separate test to check that HTTP endpoint works properly. These are different features basically

**Comment by genryxy** [comment]
@olegmoz  thanks, divided the test into two

**Review by olegmoz — APPROVED** [review]
@genryxy thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#739` is not assigned, can't get performer

**Pull request merged**